<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelSelect</class>
 <widget class="QDialog" name="LevelSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Revolutionary</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>84</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>181</width>
     <height>120</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="level1Button">
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Level One</string>
    </property>
   </widget>
   <widget class="QPushButton" name="level2Button">
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Level Two</string>
    </property>
   </widget>
   <widget class="QPushButton" name="level3Button">
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Level Three</string>
    </property>
   </widget>
   <widget class="QPushButton" name="level4Button">
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Level Four</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
